graylog2-web-interface/src/components/event-definitions/event-definition-types/FilterForm.jsx

```diff
@@ -295,6 +295,13 @@ class FilterForm extends React.Component {
             <HelpBlock>{lodash.get(validation, 'errors.execute_every_ms[0]')}</HelpBlock>
           )}
         </FormGroup>
+        <Input id="schedule-checkbox"
+               type="checkbox"
+               name="_is_scheduled"
+               label="Enable scheduling"
+               help="Whether scheduling for this event definition should be enabled?"
```

Comment:
How about:
"Should this event definition be executed?"
Or maybe don't use the word `scheduling` at all, and just call it enable/disable?

Reply:
Changed.